rts/Lua/LuaVBOImpl.cpp

```diff
@@ -499,6 +596,16 @@ size_t LuaVBOImpl::Upload(const sol::stack_table& luaTblData, sol::optional<int>
 	return UploadImpl<lua_Number>(dataVec, elemOffset, attribIdx);
 }
 
+
+/***
+ *
+ * @function VBO:Download
+ * @number[opt] attributeIndex
```

Comment:
```
* @number[opt=-1] attribIdx. If supplied with non-default value, when only data from specified attribute will be downloaded, otherwise all attributes are downloaded.
 * @number[opt=0] elementOffset. Download data starting from this element
 * @number[opt=auto] elementCount. Number of elements to download.
```